### PR TITLE
Log events

### PR DESCRIPTION
**Pull request by miiu96**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/developers/overview.md [hunk 1]
```diff
@@ -124,16 +124,44 @@ Learn about transaction's gas and how a fee is calculated:
...
+
+
+#### Log events can be categorized into the following types:
+
+| Name                                                          | Description                                                                                                                                                                                                   |
+|---------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [ESDT Operations Events](/developers/log-events/esdt-events)  | ESDT operations, which encompass token creation, transfers, and other critical actions, generate log events. These events record sender and receiver addresses, token amounts, and operation types.           |
+| [Smart Contract Call Events](/)                               | Smart contract calls often emit log events to report their execution status and results. These events typically include information such as the caller's address, the called function, and any relevant data. |
```
Comment (schimih) [inline docs/developers/overview.md:147]:
```suggestion
| [Smart Contract Call Events](/)                               | Smart contract calls often emit log events to report their execution status and results. These events typically include information such as the caller's address, the called function, and any other relevant data. |
```

Comment (miiu96) [reply docs/developers/overview.md:147]: done

File: docs/developers/log-events/esdt-events.mdx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        "AQ==",
+        "CAESAgABIvwBCAESBWx5cmFaGiAJ7NX5x1oDr20FnI7bUkyNRhecngUl6Jlvjjzlcu0BuSD0Ayo7YmFma3JlaWNqbXVwMndna21qaXR0a2dpbzZydXlpb3h6cHpraHZ6b3Jvand5eHU0Y2RiNnQycHRhaGkyQmlwZnM6Ly9iYWZrcmVpY2ptdXAyd2drbWppdHRrZ2lvNnJ1eWlveHpwemtodnpvcm9qd3l4dTRjZGI2dDJwdGFoaTpLbWV0YWRhdGE6YmFma3JlaWNqbXVwMndna21qaXR0a2dpbzZydXlpb3h6cHpraHZ6b3Jvand5eHU0Y2RiNnQycHRhaGkvbHlyYVp9"
+    ]
+}
+```
+
+</TabItem>
+</Tabs>
+
+### Semi-fungible or non-fungible or meta-esdt token transfer
```
Comment (bogdan-rosianu) [inline docs/developers/log-events/esdt-events.mdx:94]: ### Semi-fungible, non-fungible or meta-esdt token transfer

File: docs/developers/log-events/esdt-events.mdx [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+        {label: 'Example', value: 'Example'},
+    ]}
+>
+<TabItem value="Structure">
+
+| Field      | Value                                                                                                                        |
+|------------|------------------------------------------------------------------------------------------------------------------------------|
+| identifier | ESDTNFTCreate                                                                                                              |
+| address    | the creator of the token                                                                                                     |
+| topics     | `topics[0]` - token identifier base64 encoded <br/>`topics[1]` - token nonce bytes base64 encoded <br/> `topics[2]` - value bytes base64 encoded <br/> `topics[3]` - ESDigitalToken structure marshalled with gogo proto   |
```
Comment (iulianpascalau) [inline docs/developers/log-events/esdt-events.mdx:71]: with `gogo proto serializer`

File: docs/developers/log-events/esdt-events.mdx [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+For a multi-token transfer operation, multiple `MultiESDTNFTTransfer` events will be generated, one for each token being transferred.
+:::
+
+:::important Important
```
Comment (iulianpascalau) [inline docs/developers/log-events/esdt-events.mdx:182]: :+1:

File: sidebars.js [hunk 5]
```diff
@@ -124,6 +124,17 @@ const sidebars = {
           },
         ],
       },
+      {
+        type: "category",
+        label: "Log Events",
```
Comment (bogdan-rosianu) [inline sidebars.js:129]: name the category `Logs and events`
also, create an overview page for it

File: docs/developers/log-events/esdt-events.mdx [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+
+</TabItem>
+<TabItem  value="Example">
+
+```json
+{
+    "identifier": "MultiESDTNFTTransfer",
+    "address": "erd1qqqqqqqqqqqqqpgq0tajepcazernwt74820t8ef7t28vjfgukp2sw239f3",
+    "data": null,
+    "topics": [
```
Comment (bogdan-rosianu) [inline docs/developers/log-events/esdt-events.mdx:166]: in all the examples, the `topics` field if after the `data` field, but in the Structure tab the order is reversed. Please update

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**